.github/workflows/ci.yml

```
@@ -89,11 +111,21 @@ jobs:
           type=ref,event=branch
           type=ref,event=tag
 
-    - name: Set up Docker Buildx
+    - name: Set up Docker Build
```

`Buildx` is not a typo here. We are, in fact, setting up the "Buildx" builder instance here.

```suggestion
    - name: Set up Docker Buildx
```